install.sh

```diff
@@ -637,6 +637,14 @@ rm -rf "${BASE_DIR}"
 rm -rf /var/lib/kubelet
 rm -f ${BIN_DIR}/rke2
 rm -f ${KILLALL_RKE2_SH}
+
+for u in etcd rke2; do
+    if id -u \${u} 2>/dev/null; then
+        userdel \${u}
```

Comment:
If these fail for any reason they uninstall will return non-zero. Is that on purpose?
I would submit that uninstalls should rarely, if ever, exit non-zero.

Reply:
Do we want to account for that sort of edge case? The failure scnearios would be be not being able to update the passwd or group file (we're running as root) , can't remove home dir (home dir is already remvoed by this point), and user doesn't exist which we check for before attempting the removal.